Week10/app/src/main/java/com/example/marvelcharacters/view/MainActivity.kt

```diff
@@ -11,27 +11,27 @@ import androidx.work.WorkInfo
 import com.example.marvelcharacters.repository.remote.API_RESPONSE_WORKER_KEY
 import com.example.marvelcharacters.utils.toast
 import com.example.marvelcharacters.viewmodel.CharacterViewModel
+import org.koin.android.ext.android.inject
+import org.koin.androidx.viewmodel.ext.android.viewModel
 
 class MainActivity : AppCompatActivity() {
-    private val viewModel: CharacterViewModel by lazy {
-        ViewModelProviders.of(this).get(CharacterViewModel::class.java)
-    }
+    private val myViewModel: CharacterViewModel by viewModel()
```

Comment:
Don't call it `myViewModel`. In fact, don't name anything with "my...". That just looks unprofessional and doesn't provide any meaning or description.